packages/experiments/src/index.ts

```diff
@@ -5,3 +5,4 @@ export { TilesList } from './TilesList';
 export { Shimmer } from './Shimmer';
 export { Chiclet } from './Chiclet';
 export { ICustomNavLinkGroup, INavLink, NavGroupType, NavToggler } from './Nav';
+export * from './components/fluent/index';
```

Comment:
For people not on webpack 4, do you want to add a standalone importable Fluent.ts, so they don't have to pull in Shimmer/Chiclet/etc. in trying to import fluent?

Reply:
Aha! That's what those files are for. I've updated it to match the same pattern as the others, with a standalone Fluent.ts.